foreman/foreman.spec

```diff
@@ -621,6 +641,10 @@ if [ ! -e %{_datadir}/%{name}/config/initializers/encryption_key.rb -a \
   ln -s %{_sysconfdir}/%{name}/encryption_key.rb %{_datadir}/%{name}/config/initializers/
 fi
 
+# copy pregenerated apipie cache files
```

Comment:
That idea doesn't work, as we're actually going through an apipie-rails controller rather than statically serving files, so I think we'll have to copy and combine files in the rake task into the cache dir.